.github/workflows/static_analysis_and_tests.yml

```diff
@@ -38,9 +43,31 @@ jobs:
             echo "Success: Pylint score is above 9.5 (project score: $PYLINT_SCORE)."
           fi
 
+  code-format-check:
+    runs-on: ubuntu-latest
+    name: Black Format Check
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4.1.5
+
+      - name: Set up Python
+        uses: actions/setup-python@v5.1.0
+        with:
+          python-version: '3.11'
+          cache: 'pip'
+
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+
+      - name: Check code format with Black
+        id: check-format
+        run: |
+          black --check $(git ls-files '*.py')
```

Comment:
This block of code was not supposed to be there twice. Our idea is to have black tool as separate workflow, since it has different goal than pylint. Changed in: [fdafc07](https://github.com/AbsaOSS/living-doc-generator/pull/30/commits/fdafc0736596047e2a543c30bd349bbc09dd1141).

Reply:
It was my idea. There are two possible reasons for job failure.

Reply:
ok then, as the duplication was removed now, I'll approve as there is nothing else at the moment :) GJ!